Dockerfile

```diff
@@ -0,0 +1,29 @@
+# syntax=docker/dockerfile:1
+
+FROM node:19.0.0-bullseye
+ENV NODE_ENV=production
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Add runner user.
+RUN useradd -ms /bin/bash runner
+
+# Install the needed packages
+RUN apt-get update && apt-get install -y vim curl tzdata bash locales xfonts-75dpi xfonts-base
+RUN arch=$(arch | sed s/aarch64/arm64/ | sed s/x86_64/amd64/) && \
+    wget "https://github.com/jgm/pandoc/releases/download/2.19.2/pandoc-2.19.2-1-${arch}.deb" && \
+    dpkg -i pandoc-2.19.2-1-${arch}.deb && \
+    rm -rf pandoc-2.19.2-1-${arch}.deb && \
+    wget "https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6.1-2/wkhtmltox_0.12.6.1-2.bullseye_${arch}.deb" && \
+    dpkg -i wkhtmltox_0.12.6.1-2.bullseye_${arch}.deb && \
+    rm -rf wkhtmltox_0.12.6.1-2.bullseye_${arch}.deb
+
+# Install the desired npm packages
+RUN npm install --omit=dev -g @knennigtri/merge-markdown@1.0.2 doctoc@2.2.1
```

Comment:
Can we simply install the latest merge-markdown rather than 1.0.2? Or at least pull the version out and make it a variable at the top?

Reply:
Sure, I can do it. I just used fixed versions there because I tried some other combinations of their versions without any success. Perhaps it's just my experience, but I had the feeling that making them point to the latest version (either npm or apt-get) would eventually become the image unstable. I'll make all those versions configurable by docker arguments.